.github/workflows/shfmt.yml

```diff
@@ -0,0 +1,26 @@
+name: Shfmt
+
+# shfmt formats shell programs
+# https://github.com/mvdan/sh#shfmt
+
+permissions:
+  contents: read
+
+on: [workflow_dispatch, pull_request, push]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    name: Shfmt - Lint
+    runs-on: ubuntu-20.04
+    container:
+      image: mvdan/shfmt
```

Comment:
We like to pin all dependencies. Let's pin this container to its current sha

Reply:
Ahh, yes. I was thinking of that and I noticed you do it. Will do that first thing in the morning

Reply:
awesome, thank you

Reply:
Done.